### DISCUSSION TITLE
Spacing between paragraphs in quote blocks are not rendered on published site

### DISCUSSION THREAD

**Discussion by jimmytacks** [GitbookIO/community#608] [2. Published content] [answered]
I use the quote block frequently to illustrate examples of my documentation. Currently, the published version is not rendering any space between paragraphs (I'm aware from other discussions that editor and published content are not rendered the same in general).

Is this published style a design choice or a bug? If the former, I hope it can be reconsidered, and if the latter, I hope it can be fixed!

The editor view looks like this, with clear spacing between paragraphs:

![image](https://github.com/GitbookIO/community/assets/53003551/b908d96c-7ce8-4653-8768-3f863111b12a)

And the published content looks like this, with uniform spacing between both lines and paragraphs (I've tested in several browsers, all look the same):

![image](https://github.com/GitbookIO/community/assets/53003551/fd568981-b2e8-481c-843f-bf6026829317)

**Comment by aikoii** [comment, marked as answer]
Hey @jimmytacks, thank you for sharing this! We've noted it as a potential future improvement 🙂

**Comment by aikoii** [comment]
Hey @jimmytacks, just wanted to let you know we fixed that paragraph spacing issue in quote blocks 🎉
​

**Reply by jimmytacks** [reply]
Woop - legends!!! 🤩
